AcademicFlow.Managers.Contracts/IManagers/IProgramManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿using AcademicFlow.Domain.Contracts.Entities;
+using AcademicFlow.Domain.Contracts.Enums;
+using AcademicFlow.Domain.Entities;
+using AcademicFlow.Managers.Contracts.Models.ProgramModels;
+
+namespace AcademicFlow.Managers.Contracts.IManagers
+{
+    public interface IProgramManager
+    {
+        int? AddProgram(Program program);
+        void DeleteProgram(int id);
```

Comment:
why the method is not async? look at other services, it should be async and return Task